models/iterable__events.sql

```diff
@@ -1,6 +1,6 @@
 {{ config(
         materialized='incremental',
-        unique_key='event_id',
+        unique_key=['event_id','unique_user_key'],
```

Comment:
Can you elaborate on why there are two unique keys here?

Reply:
For an event-grain model in iterable, the unique key is `_fivetran_id` + `_fivetran_user_id` (in our model, `_fivetran_id` = `event_id`  and ` _fivetran_user_id` = coalesce( `unique_user_key` ,`email` ). 

In the old schema, only `_fivetran_id` (= `event_id` ) is the unique key.

But I reasoned this was still backwards compatible

Reply:
From the dbt docs, I'm not sure if there's a difference between having 2 fields here versus creating a composite key in the model to use. Is the concern that this way will affect compute more drastically?

I updated this to follow the underlying unique key changes, but haven't tested it substantially. I will need to update/add/delete records in the test data to thoroughly test it.

Previously the delete+insert strategy would affect a row with matching event_id, but now that row won't be affected unless the unique_user_key field matches that of the incoming record as well.

Reply:
Following our convo, I switched to using a composite key to keep in line with standards